lib/order.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'menu'
+
+class Order
+  attr_reader :order_placed, :balance
+
+  def initialize
+    @order_placed = []
+    @balance = 0
+  end
+
+  def add(request)
+    fail 'That item is not available' if check_menu(request) == false
```

Comment:
check_menu could be changed to a boolean output, so something like check_menu? so it will already evaluate to true/false removing need for '== false'